contrib/grinder/README.md

```diff
@@ -1,99 +1,5 @@
 # Blueflood Grinder Integration
-##Intro
-Grinder is a distributed load testing tool described [here](http://grinder.sourceforge.net/g3/getting-started.html)
 
-This code defines implementations of grinder worker threads meant to repeatedly invoke the required number of BF api calls during each "reporting interval"
+### DEPRECATED
```

Comment:
@ChandraAddala , I am not sure what you mean by the "this" in "I dont think we have to do this". Do you mean we don't have to do this whole PR?

I don't want 2 grinder implementation that we have to maintain in 2 different places. We should pick one that we want to support and go with it.